vault/provider.go

```diff
@@ -637,6 +657,21 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to configure Vault API: %s", err)
 	}
+	// Set headers if provided
+	headers := d.Get("headers").([]interface{})
```

Comment:
It would also be great to get a test where they're explicitly not provided. That probably already exists, and if so, if you could just point me to it, that'd be great.

Reply:
Yeah, I can find one - this would be new so the existing tests definitely aren't testing for this behavior.